Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -623,3 +635,11 @@ extension LoginToutViewController: ASAuthorizationControllerPresentationContextP
     return window
   }
 }
+
+// MARK: - ASWebAuthenticationPresentationContextProviding
+
+extension LoginToutViewController: ASWebAuthenticationPresentationContextProviding {
+  public func presentationAnchor(for _: ASWebAuthenticationSession) -> ASPresentationAnchor {
+    return self.view.window!
```

Comment:
Might be a good idea to unwrap this. Though, I'm don't think it's likely that window will be nil at this point

Reply:
Will do. Agreed, but perhaps better safe than sorry in this case.